authors/apps/articles/urls.py

```diff
@@ -1,10 +1,17 @@
 from django.urls import path
 
 from .views import (
-     ArticleView, ArticleRetrieveUpdateDelete
+    ArticleView, ArticleRetrieveUpdateDelete, ShareFacebookView,
+    ShareTwitterView, ShareEmailView
 )
 
 urlpatterns = [
     path('articles/', ArticleView.as_view(), name='create_article'),
     path('articles/<slug>', ArticleRetrieveUpdateDelete.as_view(), name='get_article'),
+    path('articles/<slug>/facebook',
```

Comment:
@frankopkusianwar, It's better to have the endpoint in this format `articles/<slug>/share/<platform>`.
This will give a better categorization of the endpoints